bot/cogs/tags_cog.py

```diff
@@ -107,6 +107,27 @@ async def delete(self, ctx, name):
         
         await self._delete_tag(name, ctx)
 
+    @tag.command(aliases=['info'])
+    async def about(self, ctx, name):
+
+        repo = TagRepository()
+
+        if not await repo.check_tag_exists(name, ctx.guild.id):
+            embed = discord.Embed(title= f'Error: Tag {name} does not exist', color=Colors.Error)
+            await ctx.send(embed=embed)
+            return
+
+        tag = await repo.get_tag(name, ctx.guild.id)
+
+        author = self.bot.get_user(tag['fk_UserId'])
+
+        embed = discord.Embed(title=tag['name'], color=Colors.ClemsonOrange)
```

Comment:
it would be cool to add a field for the creation date as well, we store that info so we may as well use it